src/_data/sidenav.yml

```diff
@@ -102,7 +102,7 @@
         - title: Advanced UI
           permalink: /docs/development/ui/advanced
           children:
-            - title: Actions & shortcuts
+            - title: Actions & shortcuts [NEW]
               permalink: /docs/development/ui/advanced/slivers
```

Comment:
Why does Actions & shortcuts point to slivers?

Reply:
Shouldn't it point to /docs/development/ui/advanced/actions_and_shortcuts?

Reply:
Yes!